cmake/modules/RootBuildOptions.cmake

```diff
@@ -83,6 +83,7 @@ ROOT_BUILD_OPTION(builtin_cfitsio OFF "Build the FITSIO library internally (down
 ROOT_BUILD_OPTION(builtin_xrootd OFF "Build the XROOTD internally (downloading tarfile from the Web)")
 ROOT_BUILD_OPTION(builtin_llvm ON "Build the LLVM internally")
 ROOT_BUILD_OPTION(builtin_tbb OFF "Build the TBB internally")
+ROOT_BUILD_OPTION(builtin_veccore OFF "Build VecCore internally")
```

Comment:
Nice catch, I will fix it. Vdt is also before Vc below immediately below... Further down, VecCore was added before VecGeom, so that one should be fine.